### PR TITLE
Added reference to our best practices for label naming and removed "I…

### PR DESCRIPTION
**Pull request by Alex3k**
**What this PR does / why we need it**:
Added reference to our best practices for label naming and removed "I" as this isn't a blog

### CODE REVIEW

**Comment by Alex3k** [comment]
Could someone please check that I referenced the best practices page correctly using "relref"?

File: docs/sources/fundamentals/labels.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Labels in Loki perform a very important task: They define a stream. More specifi
 
 If you are familiar with Prometheus, the term used there is series; however, Prometheus has an additional dimension: metric name. Loki simplifies this in that there are no metric names, just labels, and we decided to use streams instead of series.
 
+If in doubt, always read our [best practices]({{<relref "/best-practices">}}) when it comes to defining labels as this will hopefully save you a lot of work.
```
Comment (JStickler) [inline docs/sources/fundamentals/labels.md:20]:
```suggestion
For more tips on defining and using labels, read the [best practices]({{<relref "/best-practices">}}).
```

**Review by JStickler — APPROVED** [review]
[docs team] LGTM

**Review by jeschkies — APPROVED** [review]

**Comment by MasslessParticle** [comment]
@Alex3k Can you fix the merge conflict an the linter issue and we'll be able to merge

**Pull request closed without merging**